docs/usage/nuget.md

```diff
@@ -39,15 +39,15 @@ Alternative feeds can be specified either [in a `NuGet.config` file](https://doc
 }
 ```
 
-If this example we defined 3 NuGet feeds.
-Packages resolving will process feeds consequentially.
-It means that if package will be resolved in second feed renovate won't look in last one.
+In this example we defined 3 NuGet feeds.
+Package resolving processes feeds in the listed order, from top to bottom.
+This means that if a package is resolved in the second feed, Renovate won't check the third and final feed.
```

Comment:
Nuget datasource will use `merge` registry strategy, so will search all registries and merge the results